Showcase/Application/Core/Foundations/Assets/AssetsModel.swift

```diff
@@ -3,7 +3,6 @@
 //  Apache License 2.0
```

Comment:
Great to have succeeded in using this model in both UIKit and SwiftUI parts of the showcase.
But shouldn't we move this in Showcase/Application/Core/Foundations/Assets directory, since it is now not specific to SwiftUI anymore ?

Reply:
I totally agree with you but it's already done 🤔 I'm not sure what more I need to do.

Reply:
Sorry, I misread the name of the file in the "compare" tab of Github, I read the old file name...
You can then forget this dumb remark.